src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 1;
```

Comment:
The counter should be 0 at the beginning. Try to change logic of counter increasing